R/framework_ebp.R

```diff
@@ -7,19 +7,22 @@
 
 
 framework_ebp <- function(fixed, pop_data, pop_domains, smp_data, smp_domains,
-                          threshold, custom_indicator = NULL, na.rm, weights) {
+                          threshold, custom_indicator = NULL, na.rm,
+                          aggregate_to = NULL, weights, pop_weights) {
```

Comment:
It is ok since it is an inner function but generally, we should be careful with change of order for arguments. Again here ok since I think we always use specific arguments.